articles/aws-vs-google-cloud/index.md

```diff
@@ -0,0 +1,106 @@
+
+It also consists of offerings such as infrastructure as a service (IAAS), software as a service (SAAS), and platform as a service (PAAS). Examples of the services provided by AWS include networking, storage, remote computing, mobile development, email, analytics, and security.
+
+![Amazon Web Services](/engineering-education/aws-vs-google-cloud-services/amazon-web-services.png)
+
+[Image Source: L2L International](https://www.l2linternational.com/wp-content/uploads/2018/08/amazonaws.png)
+
+### What is Google Cloud?
+Google Cloud is a cloud computing platform that provides cloud services such as data storage, machine learning, and data analytics. It also provides offerings such as serverless computing, platform as a service, and infrastructure as a service. 
+
+Other tools offered by this platform include networking, big data, the internet of things (IoT), management tools, security, and Cloud AI.  Google cloud services share the same infrastructure with other end-user products of Google. These include YouTube, Gmail, and Google Search.
+
+![Google Cloud Platform](/engineering-education/aws-vs-google-cloud-services/google-cloud-platform.jpg)
+
+[Image Source: Software Logic](http://softwarelogic.net/wp-content/uploads/2017/07/google-cloud1.jpg)
+
+### Features of AWS
+* They have a low cost of ownership compared to dedicated or private servers.
+* The AWS platform has hybrid capabilities that give it a cutting edge over its competitors.
+* Billing and management in the AWS platform are centralized.
+* In AWS, applications can be deployed in multiple regions around the globe.
+* AWS has an introduction feature called Identity and Access Management (IAM). This feature enhances the creation of groups or users. 
+* It enhances virtual privacy through the provision of a Virtual Private Cloud (VPC) that does not interfere with other networks. 
+
+### Features of Google Cloud
+* It allows the constant inclusion of more languages.
+* In Google Cloud platforms, user experience can be improved if a better user interface is used.
+* It can enhance broad network access. 
+* Scalable services can be provided by Google Cloud platforms.
+* It allows cloud resource pooling- This is a service that enables the provider to serve many customers with provisional services. Adjustments can be made to these services to meet the demand of customers.
+  
+### Differences Between AWS and Google Cloud
+AWS and Google Cloud can be distinguished by key parameters such as users, compute services, storage, Big Data analysis, networking, features, downtimes and pricing. 
+
+#### Users
+AWS has been able to attract high-profile clients because it is established, and has extensive experience in cloud computing. AWS platform is used by clients such as Unilever, Samsung, Netflix, Xiami, BMW, and Airbnb. 

```

Comment:
I think you meant Xiaomi?

Reply:
Yes. Thanks for the correction. I have changed it.